CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: Option to change the final boss to be Arachnus, Diggernaut, Metroid Queen, or Proteus Ridley (default).
 - Changed: The DNA counter on the HUD now counts down to 0 from the amount DNA that is required to access the final boss instead of counting up to 39.
 - Changed: The automatic item tracker now shows the collected DNA out of the required DNA.
+- Fixed: Crash in Area 8 if the Baby Metroid pickup did not exist.
```

Comment:
would this ever be able to get reprod tho? i thought this was only able to get caused due to goal being set to queen, and the custom goal feature was implemented in this release.

Reply:
Hmm I guess not unless you modified the input json directly. Probably not worth mentioning then, huh?

Reply:
yeah id just remove it.